workspace_tools/targets.py

```diff
@@ -566,6 +566,23 @@ def __init__(self):
                 "template": [os.path.join(os.path.dirname(__file__), 'export', 'uvision_microlib.uvproj.tmpl')],
             }
         }
+
+class ELEKTOR_COCORICO(LPCTarget):
+    def __init__(self):
+        LPCTarget.__init__(self)
+        self.core = "Cortex-M0+"
+        self.extra_labels = ['NXP', 'LPC81X']
+        self.supported_toolchains = ["uARM", "IAR", "GCC_ARM"]
+        self.default_toolchain = "uARM"
+        self.is_disk_virtual = True
+        # ??? do I need a detect code? self.detect_code = ["1050"]
+        self.progen = {
+            "target":"CoCo-ri-Co!",  # all characters acceptable?
```

Comment:
># all characters acceptable?

I recommend using lower case, and please add this target to progen.

> # ??? do I need a detect code? self.detect_code = ["1050"]

Does this target provide mbed cmsis-dap  interface?